native/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+members = [
+    "global-key-listener",
+    "audio-recorder",
+    "text-writer",
+    "active-application",
+    "selected-text-reader",
+]
+
+resolver = "2"
+
+# Shared dependencies across workspace members can be defined here
```

Comment:
We can delete this comment. Are there other duplicated dependencies that we can move here?